cogs/matches.py

```diff
@@ -223,12 +223,23 @@ async def draw(self,ctx):
         try:
             message = await self.client.wait_for('message', timeout=30, check=lambda message: message.author == opponent and message.content.lower() == 'yes')
             self.db.delete_match(ctx.guild.id, ctx.author.id)
-            await ctx.send(embed=discord.Embed(description=f"{ctx.author.mention} {opponent.mention}, match has been draw", color=discord.Color.green()))
+            self.db.add_to_finished(match_info, "draw")
+            channel = self.client.get_channel(match_info.channel)
+            embed = discord.Embed(color=discord.Color.green())
+            pos, name, ratingChange = '', '', ''
+            players = [ctx.author,opponent]
+            for player in players:
+                pos += f"{':first_place:'}\n"
+                name += f"{player.mention}\n"
+                ratingChange += f"{self.db.get_match_rating(ctx.guild.id, player.id)[-1]}+0\n"
```

Comment:
Why is the rating change = 0 ? Even if the match is drawn, there will be some rating change.

Preferably can you make the implementation similar to [this](https://github.com/pseudocoder10/Lockout-Bot/blob/master/cogs/matches.py#L352-L375) ? Only change will be that the scores will be 0-0.

Reply:
Yes sir i will make the change and sure I will make a seperate pr for duplicate handles